### PR TITLE
debugger: Disable code if not desired

### PR DESCRIPTION
**Pull request by rzr**
This part is not as critical for TizenRT,
but will help to make iotjs smaller.

Thanks-to: Daeyeon Jeong <daeyeon.jeong@samsung.com>
Relate-to: https://github.com/Samsung/iotjs/pull/1795
IoT.js-DCO-1.0-Signed-off-by: Philippe Coval p.coval@samsung.com

### CODE REVIEW

**Comment by rzr** [comment]
Please 1st merge:
https://github.com/Samsung/iotjs/pull/1795
before:
https://github.com/Samsung/iotjs/pull/1798
Thanks

File: src/modules/iotjs_module_process.c [hunk 1]
```diff
@@ -383,10 +390,12 @@ jerry_value_t InitProcess() {
   // Set iotjs
   SetProcessIotjs(process);
   bool wait_source = false;
+#ifdef JERRY_DEBUGGER
```
Comment (LaszloLango) [inline src/modules/iotjs_module_process.c]: IMHO, this way would be better:

```c
#ifdef JERRY_DEBUGGER
  bool wait_source = false;
  if (iotjs_environment_config(iotjs_environment_get())->debugger != NULL) {
    wait_source = iotjs_environment_config(iotjs_environment_get())
                      ->debugger->wait_source;
  }

  if (!wait_source) {
    SetProcessArgv(process);
  }
#else
  SetProcessArgv(process);
#endif
```

Comment (rzr) [reply src/modules/iotjs_module_process.c]: isnt compiler figuring it out itself and strip that unused var ?

**Comment by daeyeon** [comment]
Please squash the commits into one.

File: src/modules/iotjs_module_process.c (hunk 1 above)
Comment (rzr) [reply src/modules/iotjs_module_process.c]: and this same var is used just after so I prefer to declare it false that duplicating the 2 lines

I assume compiler is smart enough to generate the same binary.

**Pull request merged**